sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -70,26 +69,39 @@ export type BeginTrainingOptions<T> = TrainModelOptions & {
 };
 
 // @public
-export type BeginTrainingWithLabelsOptions = FormRecognizerOperationOptions & {
-    prefix?: string;
-    includeSubFolders?: boolean;
-};
+export type ContentPollerLike = PollerLike<PollOperationState<RecognizeContentResultResponse>, RecognizeContentResultResponse>;
 
 // @public
-export interface CommonFieldValue {
-    boundingBox?: Point2D[];
-    confidence?: number;
-    elements?: FormElement[];
-    pageNumber?: number;
-    text?: string;
-}
+export type ContentType = "application/pdf" | "image/jpeg" | "image/png" | "image/tiff";
 
 // @public
-export type ContentPollerLike = PollerLike<PollOperationState<RecognizeContentResultResponse>, RecognizeContentResultResponse>;
+export interface CustomFormModel {
+    createdOn: Date;
+    errors?: ErrorInformation[];
+    lastModified: Date;
+    modelId: string;
+    models?: CustomFormSubModel[];
+    status: ModelStatus;
+    trainingDocuments?: TrainingDocumentInfo[];
+}
 
-// @public
-export type ContentType = "application/pdf" | "image/jpeg" | "image/png" | "image/tiff";
+// @public (undocumented)
+export interface CustomFormSubModel {
+    accuracy?: number;
+    fields: {
+        [propertyName: string]: CustomFormSubModelField;
+    };
+    formType: string;
+}
+
+// @public (undocumented)
+export interface CustomFormSubModelField {
+    accuracy?: number;
+    name: string;
+}
 
+// Warning: (ae-forgotten-export) The symbol "CommonFieldValue" needs to be exported by the entry point index.d.ts
```

Comment:
There are a handful of API Extractor warnings, all missing exports, that we should add to the index.ts exports.

Reply:
Logged https://github.com/azure/azure-sdk-for-js/issues/8384